modules/internal/SharedObject.chpl

```diff
@@ -413,13 +382,15 @@ module SharedObject {
        The result type preserves nilability of the argument type.
        If the argument is non-nilable, it must be recognized by the compiler
        as an expiring value. */
+    @deprecated(notes="shared.create from an owned is deprecated - please use a combination of :proc:`owned.release<OwnedObject.owned.release>` and :proc:`shared.adopt` instead")
```

Comment:
Let us say "please use shared.adopt instead". The part about "owned.release" should not be here. (Right??)

Also please adjust the implementation to avoid assignment.

Reply:
I put the part about `owned.release` because you cannot just do `shared.adopt(o)` and I didn't want to guide users to something that won't work. Does something like "please use shared.adopt with an unmanaged object instead" as a hint that the change requires more than just a simple renaming?